x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/QueryTranslatorTests.java

```diff
@@ -661,17 +661,17 @@ public void testTopHitsAggregationWithTwoArgs() {
 
         }
         {
-            PhysicalPlan p = optimizeAndPlan("SELECT LAST(keyword, int) FROM test");
+            PhysicalPlan p = optimizeAndPlan("SELECT LAST(date, int) FROM test");
```

Comment:
Was this change required by the types change in this PR or you just wanted some diversity in FIRST/LAST tests?

Reply:
I wanted to test the `unmapped_type` https://github.com/elastic/elasticsearch/pull/38179/files#diff-aef2b0ce456b8fdd5cc09d6cfd55f0c2R674 which before the fix was `datetime`